### DISCUSSION TITLE
Disable angle brackets on line continuation?

### DISCUSSION THREAD

**Discussion by God-damnit-all** [lzybkr/PSReadLine#3251] [Q&A] [answered]
Each new line entered in the prompt prepends the line with two 'greater than' angle brackets like so:

```powershell
PS: '
>>
>>
```

I'd like to know if there's a way to have it no longer prepend new lines like this, making it easier to copy and paste past operations into my notes.

**Comment by lzybkr** [comment, marked as answer]
You can customize your continuation prompt:

```powershell
Set-PSReadLineOption -ContinuationPrompt ''
```

Note that if you use PSReadLine to select and copy, e.x. typing <kbd>Home</kbd><kbd>Shift+End</kbd><kbd>Ctrl+c</kbd> you won't have any of the continuation prompt copied to the clipboard.
